base/libgit2/callbacks.jl

```diff
@@ -78,68 +99,40 @@ function authenticate_ssh(libgit2credptr::Ptr{Ptr{Void}}, p::CredentialPayload,
         prompt_url = git_url(scheme=p.scheme, host=p.host, username=username)
 
         # For SSH we need a private key location
-        privatekey = if haskey(ENV,"SSH_KEY_PATH")
-            ENV["SSH_KEY_PATH"]
-        else
-            keydefpath = creds.prvkey # check if credentials were already used
-            if isempty(keydefpath)
-                defaultkeydefpath = joinpath(homedir(),".ssh","id_rsa")
-                if isempty(keydefpath) && isfile(defaultkeydefpath)
-                    keydefpath = defaultkeydefpath
-                else
-                    response = Base.prompt("Private key location for '$prompt_url'",
-                        default=keydefpath)
-                    isnull(response) && return user_abort()
-                    keydefpath = unsafe_get(response)
-                end
+        if !isfile(privatekey)
+            response = Base.prompt("Private key location for '$prompt_url'",
+                default=privatekey)
```

Comment:
Might as well fix the spacing to better fit with the existing base julia style (e.g., `default=privatekey` lined up with opening parenthesis).

Reply:
I ended up leaving the indentation for the `default` keyword as is as it matches the `winprompt` indentation which is quite long.